### PR TITLE
Task/card parts confetti

### PR DESCRIPTION
**Pull request by badrinathvm**
PR provides the capability confetti with various different types , colors & intensity
confetti

### CODE REVIEW

File: CardParts/src/Extensions/UIColor.swift [hunk 1]
```diff
@@ -37,4 +37,17 @@ extension UIColor {
         let blue = CGFloat(rgbValue & 0xFF)/256.0
         return UIColor(red:red, green:green, blue:blue, alpha:1.0)
     }
+    
+    //Confetti Colors
+    static var crusta : UIColor {get{return UIColor(red: 242/255.0, green: 102/255.0, blue: 68/255.0, alpha: 1.0)}}
+    static var goldenTainoi : UIColor {get{return UIColor(red: 255/255.0, green: 198/255.0, blue: 91/255.0, alpha: 1.0)}}
+    static var monteCarlo : UIColor {get{return UIColor(red: 122/255.0, green: 198/255.0, blue: 163/255.0, alpha: 1.0)}}
+    static var strikemaster : UIColor {get{return UIColor(red: 147/255.0, green: 99/255.0, blue: 140/255.0, alpha: 1.0)}}
```
Comment (croossin) [inline CardParts/src/Extensions/UIColor.swift]: Why are some public and some not?

File: Example/CardParts/CardPartConfettiViewCardController.swift [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+//  Created by Venkatnarayansetty, Badarinath on 9/3/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import CardParts
+
+
+class CardPartConfettiViewCardController: CardPartsViewController {
+    var images: [UIImage]!
```
Comment (croossin) [inline Example/CardParts/CardPartConfettiViewCardController.swift]: Do we need to force cast these?

File: Example/CardParts/CardPartConfettiViewCardController.swift [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            let confettiView = CardPartConfettiView(frame: CGRect(x: 0, y: 150, width: self.view.bounds.width, height: self.view.bounds.height))
+            confettiView.type  = .mixed
+            confettiView.confettiImages = self.images
+            confettiView.colors = self.colors
+            confettiView.shape = CAEmitterLayerEmitterShape.line
+            confettiView.beginConfetti()
+            self.view.addSubview(confettiView)
+        
+           //change to desired number of seconds (in this case 5 seconds)
+            let when = DispatchTime.now() + 10
```
Comment (croossin) [inline Example/CardParts/CardPartConfettiViewCardController.swift]: It says 5 seconds in comment but code is `+ 10`

File: CardParts/src/Extensions/UIColor.swift (hunk 1 above)
Comment (badrinathvm) [reply CardParts/src/Extensions/UIColor.swift]: Those are not being used.I will remove them..

File: Example/CardParts/CardPartConfettiViewCardController.swift (hunk 3 above)
Comment (badrinathvm) [reply Example/CardParts/CardPartConfettiViewCardController.swift]: typo fixing it..

File: CardParts/src/Classes/Card Parts/CardPartConfettiView.swift [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    public var colors:[UIColor] = [ UIColor.red, UIColor.green, UIColor.blue ]
+    public var type:ConfettiType = .star
+    /// default : 0.5 value ranges from 0 - 1(being very slow)
+    public var intensity:Float = 0.5
+    public var shape:CAEmitterLayerEmitterShape = .sphere {
+        didSet {
+            emitter.emitterShape = shape
+        }
+    }
+    public var confettiImages:[UIImage] = [UIImage(named: "confetti", in: Bundle(for: CardPartConfettiView.self),compatibleWith: nil)!]
```
Comment (croossin) [inline CardParts/src/Classes/Card Parts/CardPartConfettiView.swift]: I'd prefer if we did not forcecast here

File: CardParts/src/Extensions/UIColor.swift (hunk 1 above)
Comment (badrinathvm) [reply CardParts/src/Extensions/UIColor.swift]: Fixed

File: Example/CardParts/CardPartConfettiViewCardController.swift (hunk 3 above)
Comment (badrinathvm) [reply Example/CardParts/CardPartConfettiViewCardController.swift]: `guarded` it

File: CardParts/src/Classes/Card Parts/CardPartConfettiView.swift (hunk 4 above)
Comment (badrinathvm) [reply CardParts/src/Classes/Card Parts/CardPartConfettiView.swift]: Fixed.

File: Example/CardParts/CardPartConfettiViewCardController.swift (hunk 2 above)
Comment (badrinathvm) [reply Example/CardParts/CardPartConfettiViewCardController.swift]: Fixed

**Pull request merged**